src/GeositeFramework/js/Plugin.js

```diff
@@ -461,6 +468,24 @@ require(['use!Geosite',
             return view;
         }
 
+        function addInfographicButton(view, infographicConfig, sourceFolder) {
+            view.$uiContainer.append(N.app.templates['infographic-button-template']());
+            view.$uiContainer.find('.nav-title').addClass('title-with-graphic');
+            view.$uiContainer.find('.infographic-icon').on('click', function() {
+                showInfographic(infographicConfig, sourceFolder);
+            });
+        }
+
+        function showInfographic(infographicConfig, sourceFolder) {
+            TINY.box.show({
+                animate: true,
+                url: sourceFolder + '/html/infographic.html',
+                fixed: true,
+                width: infographicConfig[0],
+                height: infographicConfig[1]
```

Comment:
Should we add guards or defaults or throws in case `infographicConfig` is not null but also not a list with at least two elements?

Reply:
Added!